lib/getContractSources.js

```diff
@@ -5,21 +5,46 @@ import path from 'path'
  * Extract contents of contract files as string
  *
  * @param  {Array}  contracts             Contracts to compile
+ * @param  {String} modulesPath           Path to node_modules directory
  * @param  {String} inputPath             Directory to read target contracts from
  * @param  {String} contractFileExtension File extension of the target contracts
  * @return {Array}                        Promises resolving contract file content
  */
-const getContractSources = (contracts, inputPath, contractFileExtension) => {
+const getContractSources = (
+  contracts,
+  modulesPath,
+  inputPath,
+  contractFileExtension
+) => {
   return contracts.map(async contract => {
     try {
+      // redeclare injections to allow for them to overriden
+      let filePath = inputPath
+      let contractFilename = contract
+
+      // Check for contract dependencies installed via npm
+      if (contractFilename.includes('/')) {
+        // Split the desired import path
+        const pathParts = contractFilename.split('/')
```

Comment:
This needs to be flexible enough to accommodate different contract paths, for example:
- `contracts/One.sol`
- `contracts/Two/Two.sol`
- `node_modules/zeppelin-solidity/contracts/token/BasicToken.sol`

Reply:
The below pop method returns the last item in the split array, that being the filename. So it handles any length of directory structure. In respect to handling the `node_modules` directory, that is handled above.

Reply:
The problem is the local nested contract `contracts/Two/Two.sol` would be caught by the conditional and processed as a remote contract.

Reply:
Yup, fixed that locally. But putting greater thought toward the general approach. Will push update in due course.